### PR TITLE
Speciesboxes; species specific starting boxes

### PR DESCRIPTION
**Pull request by Birdtalon**
This PR introduces species specific starting boxes.

This will allow Vox to have N2 tanks instead of O2, plasmamen to have plamsa etc.

🆑 Birdtalon
add: Species specific starting boxes.
/🆑

### CODE REVIEW

**Comment by FlattestGuitar** [comment]
yooooooo

https://nanotrasen.se/forum/topic/9238-suggestion-fix-okit

File: code/datums/starting_boxes.dm [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+// OUTFIT STARTING BOX DATUMS
+
+/datum/starting_box/outfit
+    name = "generic outfit starting box"
+    boxtype = STARTING_BOX_OUTFIT
+
+/datum/starting_box/outfit/response_team
+    name = "ert starting box"
+    boxtype = STARTING_BOX_OUTFIT
```
Comment (Birdtalon) [inline code/datums/starting_boxes.dm]: Already inherits this.

**Comment by tigercat2000** [comment]
The whole /datum/starting_box thing seems like an over the top solution to something that could be implemented in a much less abstract way by say, making a number of preset species boxes and making a subtype that spawns the correct one in it's place by looking at the user species.

**Comment by KasparoVy** [comment]
vOx FiNaLlY gEt BoX :bird: :100:

**Comment by FreeStylaLT** [comment]
Box for Vox Pox

**Comment by Birdtalon** [comment]
Scrapped my idea in favour of Tigercat's simpler one.

**Comment by Spartan6** [comment]
[N2 Box](https://cdn.discordapp.com/attachments/361959244918030367/514540212571865109/unknown.png) and [Plasma Box](https://cdn.discordapp.com/attachments/361959244918030367/514541812162101279/unknown.png) sprites in case you are interested.

**Comment by variableundefined** [comment]
@Birdtalon May I ask who made those sprites and what licenses are they under?

**Pull request merged**